WordPress/src/main/java/org/wordpress/android/ui/bloggingprompts/onboarding/OnboardingDialogFragment.kt

```diff
@@ -0,0 +1,63 @@
+package org.wordpress.android.ui.bloggingprompts.onboarding
+
+import android.app.Dialog
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.view.Window
+import androidx.fragment.app.DialogFragment
+import androidx.lifecycle.ViewModelProvider
+import org.wordpress.android.R
+import org.wordpress.android.R.attr
+import org.wordpress.android.WordPress
+import org.wordpress.android.databinding.BloggingPromptsOnboardingDialogFragmentBinding
+import org.wordpress.android.util.getColorFromAttribute
+import org.wordpress.android.util.isDarkTheme
+import javax.inject.Inject
+
+class OnboardingDialogFragment : DialogFragment() {
```

Comment:
This is very minor, but what do you think about naming this `BloggingPromptsOnboardingDialogFragment` just to be a bit more specific?